plugin/src/main/java/com/yelp/codegen/KotlinGenerator.kt

```diff
@@ -205,8 +205,13 @@ open class KotlinGenerator : SharedCodegen() {
     @VisibleForTesting
     internal fun addRequiredImports(codegenModel: CodegenModel) {
```

Comment:
Please make sure you write tests for this method. There are already some inside the KotlinGeneratorTest.kt just add cases for the `JsonClass`. 

Also adding a test would document why you changed from `vars` to `allVars`

Reply:
The change of `codegenModel.vars.isNotEmpty()` to `codegenModel.allVars.isNotEmpty()` has been done to make the code more consistent as in line 219 we're iterating over `codegenModel.allVars`.

`vars` and `allVars` are equivalent while `supportsInheritance` is set to `false` (current state of the library).
>      // By default these are the same collections. Where the code generator supports inheritance, composed models
>      // store the complete closure of owned and inherited properties in allVars and allMandatory.
>      allVars = vars;

Reply:
Gotcha. The test request is still valid anyway.

Reply:
Sure ;)
They are there already.

Just wanted to briefly mention why I've modified it ;)